packages/core/src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,78 @@
+import { h, Component, ComponentInterface, Event, EventEmitter, Prop } from '@stencil/core';
+
+import { PaginationSelectPageEvent } from './pagination.interfaces';
+
+@Component({
+  tag: 'dso-pagination',
+  styleUrl: 'pagination.scss',
+  shadow: true,
+})
+export class Pagination implements ComponentInterface {
+  /**
+   * Total pages
+   */
+  @Prop()
+  count!: number;
+
+  /**
+   * Current page
+   */
+  @Prop()
+  current!: number;
+
+  /**
+   * Current page
+   */
+  @Prop()
+  label!: string;
+
+  /**
+   * Function is called to construct the href for each anchor element.
+   */
+  @Prop()
+  createLink!: (page: number) => string;
+
+  /**
+   * Emitted on page select
+   */
+  @Event()
+  selectPage!: EventEmitter<PaginationSelectPageEvent>;
+
+  clickHandler = (e: MouseEvent, page: number) => {
+    e.preventDefault();
+
+    this.selectPage.emit({ originalEvent: e, page });
+  };
+
+  render() {
+    const pages = Array.from({ length: this.count }, (_value, i) => i + 1);
+
+    return (
+      <nav aria-label={this.label}>
```

Comment:
`<nav>` uit het component tillen, we zijn tot de conclusie gekomen dat dit niet het begin van Pagination is maar dat deze semantiek bij de afnemer ligt.